src/components/radio/Radio.stories.tsx

```diff
@@ -33,3 +33,13 @@ export const Colored = Template.bind({});
 Colored.args = {
   colored: true,
 };
+
+export const Small = Template.bind({});
+Small.args = {
+  small: true,
+};
+
+export const Disabled = Template.bind({});
```

Comment:
Could you add `disabled` as a prop on the `default` story? and then remove this `Disabled` story? That way we can control all props in the controls and don't have to switch to different stories